Source/Plugins/Sampling/Documentation/SamplingFilters/SampleSurfaceMeshSpecifiedPoints.md

```diff
@@ -0,0 +1,71 @@
+SampleSurfaceMeshSpecifiedPoints {#samplesurfacemeshspecifiedpoints}
+=====
+
+## Group (Subgroup) ##
+Sampling (Resolution)
+
+## Description ##
+This Filter 'samples' a triangulated surface mesh with a specified list of **Vertices** (or points) read from a file.  The sampling is performed by the following steps:
+
+1. Determine the bounding box and **Triangle** list of each **Feature** by scanning all **Triangles** and noting the **Features** on either side of the **Triangle**.
+2. For each **Vertex** read from the file, determine which bounding box(es) they fall in (*Note:* the bounding box of multiple **Features** can overlap).
+3. For each bounding box a **Vertex** falls in, check against that **Feature's** **Triangle** list to determine if the **Vertex** falls within that n-sided polyhedra (*Note:* if the surface mesh is conformal, then each **Vertex** will only belong to one **Feature**, but if not, the last **Feature** the **Vertex** is found to fall inside of will *own* the **Vertex**).
+4. Assign the **Feature** number that the **Vertex** falls within to the *Feature Ids* array in the new **Vertex** geometry.
+
+The filter will write out a file with the list of *FeatureIds* for the **Vertices**.  The filter also creates a new *DataContainer* to hold the **Vertex** geometry and the *FeatureIds* that live on them.
+
+The format of the *Specified Points File* is as follows:
+
+~~~~~~~~~~~~~~~
+Number of Points
+X, Y, Z
+X, Y, Z
+X, Y, Z
+.
+.
+.
+X, Y, Z
+~~~~~~~~~~~~~~~
+
+
+## Parameters ##
+| Name | Type | Description |
+|------|------|------|
+| Specified Points File Path | File Path | Location of file listing points to sample at |
+| Sampled Values File Path | File Path | Location where output information will be written |
+
+## Required Geometry ##
+Triangle
+
+## Required Arrays ##
+| Type | Default Name | Type | Component Dimensions | Description |
+|------|--------------|-------------|---------|-----|
+| Triangle | SurfaceMeshFaceLabels | Int | (2) | Specifies which **Features** are on either side of each **Triangle**. |
+
+## Created Arrays ##
+| Type | Default Name | Type | Component Dimensions | Description |
+|------|--------------|-------------|---------|-----|
+| Cell | FeatureIds | Int | (1) | Specifies to which **Feature** each **Cell** belongs. |
+
+
+## Authors ##
+
+**Contact Info** dream3d@bluequartz.net
+
+**Version** 1.0.0
+
+**License**  See the License.txt file that came with DREAM3D.
+
```

Comment:
Remove the "Authors" section, as we have decided to keep this out of the filter documentation.